docs/source/developer_guide/workflows.rst

```diff
@@ -31,13 +31,13 @@ To do so place in your ``__init__.py`` file in the main workflow directory
 	
 	    return BaseFactory(module, Workflow, "aiida.workflows.user.epfl_theos")
 	
-In this example, a workflow located in e.g. ``aiida/workflows/user/epfl_theos/quantumespresso/pw.py``
+In this example, a workflow located in e.g. ``aiida/workflows/user/epfl_theos/foldername/plugin.py``
 can be loaded simply by typing::
 	
-	TheosWorkflowFactory('quantumespresso.pw')
+	TheosWorkflowFactory('foldername.plugin')
```

Comment:
Should we keep the 'Theos' reference in the `Factory` as well as above in the folder? We can just use `aiida/workflows/user` as the base name which would also work

Reply:
I'll remove the Theos reference - I'll still keep the subfolder structure because I think it's better organisation. I'll also do a couple of improvements mentioning the new plugin system.